### PR TITLE
Update access-point.md

### PR DESCRIPTION
**Pull request by SurferTim**
Edit to conform to Jessie and Stretch releases.

### CODE REVIEW

**Comment by SurferTim** [comment]
Found two errors.
Change this
ssh pi@192.168.0.1
to this:
ssh pi@192.168.4.1

And change this to manual in /etc/network/interfaces:
```
# Bridge setup
auto br0
iface br0 inet manual
bridge_ports eth0 wlan0

```

**Comment by JamesH65** [comment]
You can simply change the PR by editing the branch with your existing
changes, all changes automatically go to the PR.

On 25 September 2017 at 14:03, SurferTim <notifications@github.com> wrote:

> Found two errors.
> Change this
> ssh pi@192.168.0.1
> to this:
> ssh pi@192.168.4.1
>
> And change this to manual in /etc/network/interfaces:
>
> # Bridge setup
> auto br0
> iface br0 inet manual
> bridge_ports eth0 wlan0
>
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_raspberrypi_documentation_pull_735-23issuecomment-2D331873888&d=DwMFaQ&c=DpyQ_ftY536pf7wCBQXXU58xADDRY77THQzJu1OmzOo&r=w09_2ePv8G3zRjoV19Wm1Q6rI7CDlOns4PuRv2hHkek&m=sMXv8jKEuqZo717CaFVDzYVDX1w0p-X90i9QvPLWV14&s=DmQo3cH3H05kxMeddsEC0dTzUxqeqT2rKE3WC9bGoZg&e=>,
> or mute the thread
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_ADqrHYayOw8bnITXifMubfCSpc-5F2cXIxks5sl6SxgaJpZM4PiCdf&d=DwMFaQ&c=DpyQ_ftY536pf7wCBQXXU58xADDRY77THQzJu1OmzOo&r=w09_2ePv8G3zRjoV19Wm1Q6rI7CDlOns4PuRv2hHkek&m=sMXv8jKEuqZo717CaFVDzYVDX1w0p-X90i9QvPLWV14&s=sTDGqnc6ekBPPZu5QftqLg2LPWcvP6GuHLxgcuHfrfI&e=>
> .
>



-- 
James Hughes
Principal Software Engineer,
Raspberry Pi (Trading) Ltd

**Comment by SurferTim** [comment]
@JamesH65: Thanks. i don't use Github as a contributor much, as you can probably tell. Thanks for the tip, and the changes have been made.

File: configuration/wireless/access-point.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ In order to work as an access point, the Raspberry Pi will need to have access p
 Use the following to update your Raspbian installation:
 ```
 sudo apt-get update
-sudo apt-get dist-upgrade
+sudo apt-get upgrade
```
Comment (XECDesign) [inline configuration/wireless/access-point.md]: Why?

File: configuration/wireless/access-point.md [hunk 2]
```diff
@@ -70,10 +55,10 @@ Type or copy the following information into the dnsmasq configuration file and s
 
 ```
 interface=wlan0      # Use the require wireless interface - usually wlan0
-  dhcp-range=192.168.0.2,192.168.0.20,255.255.255.0,24h
+  dhcp-range=192.168.4.2,192.168.4.20,255.255.255.0,24h
 ```
```
Comment (JamesH65) [inline configuration/wireless/access-point.md]: Finally got a chance to look at this. Just out of interest, why did you change this to .4?

Comment (SurferTim) [reply configuration/wireless/access-point.md]: I changed it because many routers use 192.168.0.x/24 or 192.168.1.x/24 for the localnet. The AP subnet must be different.

Comment (SurferTim) [reply configuration/wireless/access-point.md]: Note that the IP address of the wlan interface was also changed to 192.168.4.1/24 in /etc/dhcpcd.conf. The subnets must match in these two locations.

**Comment by JamesH65** [comment]
Sorry, still not had time to go through this to ensure its technically correct.  I've not forgotten!

**Comment by SurferTim** [comment]
No problem. I may have another contribution to the page. I'm testing an access point using one wireless interface for both the ap and the client. I'll wait until this is approved first.

File: configuration/wireless/access-point.md [hunk 3]
```diff
@@ -27,35 +27,20 @@ sudo systemctl stop hostapd
...
+    static ip_address=192.168.4.1/24
 
 Now restart the dhcpcd daemon and set up the new `wlan0` configuration:
 
 ```
 sudo service dhcpcd restart
-sudo ifdown wlan0
-sudo ifup wlan0
-```
+``
```
Comment (MattAKAFred) [inline configuration/wireless/access-point.md]: minor typo here -  

"Now restart the dhcpcd daemon and set up the new `wlan0` configuration:" is inside the code block, 
"sudo service dhcpcd restart" is not

**Comment by JamesH65** [comment]
Sorry, still not had time to look at this - been very busy.

**Comment by zamai** [comment]
Hey guys! I was struggling to setup new Pi3B as an access point following, was following official guide, but as I understood it is written for the previous version of Raspbian.
Once I this PR, I've followed it exactly and it worked as a charm!
Please merge this pr so that new people are seeing updated docs on the official website.
Thanks

**Comment by JamesH65** [comment]
That's good to know, I've simply not had the time to check the PR for accuracy, so useful to know it does work!

**Comment by SurferTim** [comment]
Checked with Stretch 2017-11-29. Works great.

**Comment by JamesH65** [comment]
OK, since this has been independently checked, I'm going to push it to Copy edit for inclusion in the docs. I've just not had time to check it myself.
